tests/pages/firefox/whatsnew.py

```diff
@@ -5,7 +5,7 @@
 from selenium.webdriver.common.by import By
 
 from pages.firefox.base import FirefoxBasePage
-from pages.regions.send_to_device import SendToDevice
```

Comment:
When running the whatsnew tests locally, tests are failing with the following error:

```
E       AttributeError: 'FirefoxWhatsNewPage' object has no attribute 'send_to_device'
```

This is becuase [this test](https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/test_whatsnew.py#L13) is still using the old send to device widget, and it expects the `SendToDevice` object to be present for the test to run.

This PR doesn't need to replace `SendToDevice` with `SendYourself` just yet, since we haven't deprecated the old widget. All you should need here is add `SendYourself` and then use that in the [Indonesia whats new test](https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/test_whatsnew.py#L32).